### PR TITLE
updated details for Azure Singapore event

### PR DESCRIPTION
**Pull request by NileshGule**
Added folder for Global Azure Bootcamp Singapore event

### CODE REVIEW

**Review by noopman — CHANGES_REQUESTED** [review]
Hello there @NileshGule and thank you for your community submission! There are a few items you need to attend to here to make it a complete submission. The logo file should be a community logo. If you do not have one, just make a simple image with your community name. Please refer to the other [communities](https://globalazure.net/communities/2023) already submitted for reference. Also we need a markdown file (.md) with your community intro! Your PR should have three files. This submission only has two and the image is not a logo! Check the [readme](https://github.com/globalazure/communities#readme) for complete instructions!

**Comment by noopman** [comment]
Pleaes advise @NileshGule!

**Comment by NileshGule** [comment]
I’ll update the PR soon with suggested changes

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

File: 2023/AzureSingapore/data.json [hunk 1]
```diff
@@ -0,0 +1,30 @@
+{
+  "ValidMaintainers": {
+    "Description": "GitHub users who maintain this folder. PR for changing this folder will only be accepted from these users.",
+    "GitHubHandles": [
+      "nileshgule",
+      "",
+      ""
```
Comment (noopman) [inline 2023/AzureSingapore/data.json:7]: You can also remove empty items here!

**Comment by NileshGule** [comment]
@noopman I have made the requested updates. Pls review and let me know if any other changes are required

File: 2023/AzureSingapore/data.json [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    "Description_SessionizeApiId": "https://sessionize.com/api/v2/10616/view/Sessions",
+    "SessionizeEventName": "https://sessionize.com/global-azure-bootcamp-singapore-2023/",
+    "Logo": {
+      "Description": "Azure Singapore Community Logo",
+      "File": "Azure-Singapore.png"
+    },
+    "Locations": [
+      {
+        "Name": "Global Azure Bootcamp Singapore",
+        "Address": "{Optional address of the location}",
```
Comment (OlenaBorzenko) [inline 2023/AzureSingapore/data.json:22]: I see the latitude and longitude are provided, is it correct that addresses are not set? It's optional but just making sure.

File: 2023/AzureSingapore/data.json [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+    "Description": "GitHub users who maintain this folder. PR for changing this folder will only be accepted from these users.",
+    "GitHubHandles": [
+      "nileshgule"
+    ]
+  },
+  "CommunityInfo": {
+    "Name": "Azure Singapore",
+    "Country": "Singapore",
+    "Url": "https://www.meetup.com/en-AU/mssgug/events/291424203",
+    "SessionizeApiId": "{Your sessionize event Id}",
```
Comment (OlenaBorzenko) [inline 2023/AzureSingapore/data.json:12]: ApiId is missing, could you please provide this value?

**Review by OlenaBorzenko — CHANGES_REQUESTED** [review]

File: 2023/AzureSingapore/data.json (hunk 2 above)
Comment (noopman) [reply 2023/AzureSingapore/data.json:22]: @NileshGule is it the case that you don't know the street address of your event just yet?

Comment (NileshGule) [reply 2023/AzureSingapore/data.json:22]: @noopman Yes we are still finalizing the venue. In the past we used to conduct the Singapore Global Azure Bootcamp in the Microsoft office. This year we might have some logistic issues, so we are still in discussion with folks from Microsoft Singapore.

File: 2023/AzureSingapore/data.json (hunk 3 above)
Comment (NileshGule) [reply 2023/AzureSingapore/data.json:12]: @OlenaBorzenko @noopman API ID has been updated. please review the latest changes.

Comment (noopman) [reply 2023/AzureSingapore/data.json:12]: Sorry @NileshGule, but I do not see your latest changes here in the PR. I can only see the text "{Your sessionize event Id}" for the API ID.

Comment (noopman) [reply 2023/AzureSingapore/data.json:12]: The reason for us being picky is to be helpful and ensure your event is represented as intended on Global Azure! @NileshGule

Comment (noopman) [reply 2023/AzureSingapore/data.json:12]: When I go to here [https://github.com/globalazure/communities/commit/d71440b0bb6f54b075c40f7411875785c73d5c55](https://github.com/globalazure/communities/commit/d71440b0bb6f54b075c40f7411875785c73d5c55) It says "This commit does not belong to any branch on this repository, and may belong to a fork outside of the repository."

**Review by noopman — APPROVED** [review]
There! Done! Thank you for being so dilligent! Welcome to Global Azure!

**Pull request merged**